jobs.groovy

```diff
@@ -427,6 +438,33 @@ def build_steps(job) {
 }
 
 
+/*
+    Closure that configures the current project to notify slack on failure.
+    The specified channel will be notified.
+*/
+Closure notify_slack(String channel) {
+    return {
+        // I'm not sure how to put these magic strings in a variable
+        // or avoid them altogether.
+        it / 'properties' << 'jenkins.plugins.slack.SlackNotifier_-SlackJobProperty'(plugin: 'slack@1.8.1') {
+            room(channel)
```

Comment:
The fix was here, it needed to use the parameter. It was working before
because it was accidentally referencing a global. When I merged with
changes in master the global was no longer present and it failed.